### ISSUE TITLE
@apply sm: is not generated with npm run hot/watch

### ISSUE THREAD

**Issue by Livijn** [tailwindlabs/tailwindcss-jit#132]
### What version of @tailwindcss/jit are you using?

0.1.4

### What version of Node.js are you using?

15.11.0

### What browser are you using?

Chome

### What operating system are you using?

macOS

### Reproduction repository

lul

I am missing classes when using sm modifiers together with `@apply`. The media query seems to be generated but without any content. This does not happen when running `npm run prod`. I am just using `mix.sass('resources/sass/app.scss', 'public/css')`.

SCSS:
```css
@layer components {
  .btn-green, .btn-indigo, .btn-red, .btn-orange, .btn-white, .btn-gray {
    @apply border font-bold rounded-md py-2 px-3 sm:py-3 sm:px-4 focus:ring-2;

    &:disabled {
      @apply opacity-50 cursor-not-allowed;
    }
  }
}
```

CSS:
```css
.btn-green, .btn-indigo, .btn-red, .btn-orange, .btn-white, .btn-gray{
	border-radius: 0.375rem;
	border-width: 1px;
	padding-left: 0.75rem;
	padding-right: 0.75rem;
	padding-top: 0.5rem;
	padding-bottom: 0.5rem;
	font-weight: 700;
}
@media (min-width: 640px){}
.btn-green:disabled, .btn-indigo:disabled, .btn-red:disabled, .btn-orange:disabled, .btn-white:disabled, .btn-gray:disabled{
	cursor: not-allowed;
	opacity: 0.5;
}
```

tailwind.config.js
```js
module.exports = {
  purge: {
    content: [
      './resources/js/**/*.vue',
      './resources/sass/*.scss',
      './resources/views/**/*.blade.php',
    ],
    options: {
      safelist: [/md:grid-cols-\d+/, /lg:grid-cols-\d+/]
    }
  },
  // etc
}
```

**Comment by jessedobbelaere** [comment]
It sounds like I have the same issue but with an `lg:hidden` class used with `@apply`. I'm trying to isolate the issue from my custom CMS setup into a simple repo using HTML and webpack-dev-server but haven't managed to reproduce it in a new repo. In the meantime, here are some findings. I use webpack `5.27.1`, PostCSS `8.2.8` and `@tailwindcss/jit: 0.1.5`. 

<details>
<summary>My findings so far</summary>

app.css only contains:
```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer components {
    .hamburger {
        @apply block lg:hidden;
    }
}
```

My HTML code

```html
<header class="c-header">
    <button class="js-main-nav-trigger hamburger hamburger--spin" type="button" aria-label="Menu" aria-controls="navigation">
        MENU BUTTON
    </button>

    <div class="lg:hidden">
        Mobile menu goes here
    </div>
</header>
```

So to summarize, we have a mobile menu (with `lg:hidden` in the HTML code) and a hamburger menu on which I use `lg:hidden` with the `@apply` method. When I render this locally, I can see the `lg:hidden` class is working on the mobile menu, but it did not set it on the `.hamburger` class using `@apply`. Weird that the mediaquery is set, but it's empty. This is the same thing that @Livijn found. It used to work fine on webpack-dev-server with regular TailwindCSS, but not yet with the jit compiler. 

<img width="948" alt="image" src="https://user-images.githubusercontent.com/1352979/111913206-65dd4380-8a6d-11eb-860a-c1f1ae014898.png">
</details>

Here's a video to illustrate how weird it behaves: 

https://user-images.githubusercontent.com/1352979/111925190-19632980-8aa8-11eb-8882-ad391c8b7a62.mp4

Will try to make a reproduction repository... 🕵️

**Comment by adamwathan** [comment]
This should be fixed in v0.1.6, thanks!

**Comment by jessedobbelaere** [comment]
Awesome, tested and confirming that it works on v0.1.6! 👍 🎉
